pkg/base1/cockpit.js

```diff
@@ -1757,6 +1757,11 @@ function full_scope(cockpit, $, po) {
             ensure_cache();
             return new DBusProxies(self, cache, String(iface), String(path_namespace));
         };
+
+        self.update_props = function update_props(path, iface, props) {
```

Comment:
I would suggest:
 * Having this trigger the notify event correctly.
 * Using the same input data structure here as with msg.notify and the notify event.
 * Calling this method ```.notify()```